setup.py

```diff
@@ -24,14 +24,14 @@
     packages=find_packages(exclude=['tests']),
     include_package_data=True,
     install_requires=[
-        'pyspark',
-        'python_moztelemetry',  # TODO: pin version
-        'click',
-        'requests',
-        'click_datetime',
-        'arrow',
-        'scipy',
-        'numpy'
+        'arrow==0.10.0',
+        'click==6.7',
+        'click_datetime==0.2',
+        'numpy==1.13.3',
+        'pyspark==2.2.0',
```

Comment:
I've tested this locally using `tox -r`, and it seems like I get the same error. Running `pip freeze | grep pyspark` gives `pyspark==2.2.0`, but tox is expecting `2.2.0.post0`. Possibly a packaging error on Spark's side?